.golangci.yaml

```diff
@@ -51,10 +55,23 @@ linters-settings:
   goimports:
     local-prefixes: "github.com/openfga/openfga"
   importas:
-    # Do not allow unaliased imports of aliased packages.
-    no-unaliased: true
-    # Do not allow non-required aliases.
-    no-extra-aliases: false
+    no-unaliased: true # Do not allow unaliased imports of aliased packages.
+    no-extra-aliases: false # Do not allow non-required aliases.
     alias:
       - pkg: github.com/openfga/api/proto/openfga/v1
         alias: openfgav1
+
+issues:
+  exclude-use-default: false
+  exclude-rules:
+    - path: "(.+)_test.go"
+      linters:
+        - bodyclose
+  exclude:
+    - "should have a package comment" # stylecheck | revive
+    - "exported (.+) should have comment" # revive
+    - "comment on exported (.+) should be of the form" # revive
+    - "(.+) name will be used as (.+) by other packages, and that stutters; consider calling this (.+)" # revive
+    - "Error return value of (.+) is not checked" # errcheck
+    - "unused-parameter" # revive
+    - "exported func (.+) returns unexported type (.+), which can be annoying to use" # revive
```

Comment:
Ignoring these for now, as they need to be tackled in dedicated PRs due to the high volume of issues.